styles.css

```diff
@@ -0,0 +1,323 @@
+.top {
+    display: flex;
+
+    height: 65px;
+    background: #FFFFFF;
+    box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.16);
+    /* justify-content: space-around; */
+    /* margin-bottom: 52px; */
+
```

Comment:
1. why are there spaces between rules? make your code look nice and consistent
2. remove comments. there should be no code that is not used